applications/PfemFluidDynamicsApplication/python_scripts/remesh_fluid_domains_process.py

```diff
@@ -230,6 +230,9 @@ def ExecuteInitializeSolutionStep(self):
                 node.SetSolutionStepValue(KratosMultiphysics.FLUID_FRACTION,1.0)
                 node.SetSolutionStepValue(KratosMultiphysics.FLUID_FRACTION_OLD,1.0)
                 node.SetSolutionStepValue(KratosMultiphysics.FLUID_FRACTION_RATE,0.0)
+        if(currentStep == 2):
+            for node in self.main_model_part.Nodes:
```

Comment:
minor suggestion for improvement, there is a fct for this in the VariableUtils

Reply:
Thanks for the tip. I made the corresponding changes here, although they must be implemented in many other places too. I am approving the pull request